core/src/main/java/com/datastrato/gravitino/authorization/AccessControlManager.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.authorization;
+
+import com.datastrato.gravitino.EntityStore;
+import com.datastrato.gravitino.exceptions.NoSuchUserException;
+import com.datastrato.gravitino.exceptions.UserAlreadyExistsException;
+import com.datastrato.gravitino.storage.IdGenerator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * AccessControlManager is used for manage users, roles, grant information, this class is an
+ * entrance class for tenant management.
+ */
+public class AccessControlManager {
+
+  private static final Logger LOG = LoggerFactory.getLogger(AccessControlManager.class);
+
+  private final UserManager userManager;
+
+  /**
+   * Constructs a AccessControlManager instance.
+   *
+   * @param store The EntityStore to use for managing access control.
+   * @param idGenerator The IdGenerator to use for generating identifiers.
+   */
```

Comment:
I think we can remove some of the javadoc here, it's not so useful.

Reply:
done. Removed the comment.